BOLD-04.md

```diff
@@ -1,234 +1,113 @@
-# BOLD-04
+# BOLD #4: Swap Protocol
 
-## Swap Protocol
+Once an order match is found on the taker order book, the following swap procedure is expected instantiate. The following is the protocol happy flow: 
```

Comment:
better wording: "the following swap procedure is expected to execute"

Reply:
Please check the new version.